src/core/software/pupil-player.md

```diff
@@ -217,6 +217,57 @@ Toggle `Show fixations` to show a visualization of fixations. The blue number is
 
 You can find more information in our [dedicated fixation detection section](/core/terminology/#fixations "Pupil Core terminology - fixations").
 
+#### Blink Detector
+Pupil Core's Blink Detector leverages the fact that [2D pupil confidence](/core/terminology/#confidence) drops rapidly 
+in both eyes during a [blink](/core/terminology/#blinks) as the pupil becomes obscured by the eyelid, followed by a rapid 
+rise in confidence as the pupil becomes visible again.
+
+The Blink Detector processes [2D pupil confidence](/core/terminology/#confidence) values by convolving them with a 
+[filter](https://github.com/pupil-labs/pupil/blob/eb8c2324f3fd558858ce33f3816972d93e02fcc6/pupil_src/shared_modules/blink_detection.py#L360). 
+The filter response – called 'activity' – spikes the sharper the confidence drop is and *vice versa* for confidence 
+increases. 
+
+Blinks are subsequently detected based on onset and offset confidence thresholds and a filter length in seconds.
+
+- **Onset Confidence Threshold:** The threshold that the filter response ('Activity') must rise above to classify the
+  onset of a blink, corresponding to a sudden **drop** in 2D pupil detection confidence
+- **Offset Confidence Threshold:** The threshold that the filter response ('Activity') must fall below to classify the 
+  end of a blink, corresponding to a **rise** in 2D pupil detection confidence
+- **Filter Length:** The time window's length in which the detector tries to find confidence drops and gains
+
+In Pupil Player, the green data series labelled 'Activity' is the filter response, and the yellow bars that denote 
+onset and offset confidence are above and below this, respectively. The blink detection result is shown by the red 
+line, characterised by a step function where blinks have been classified.
+
+<div class="pb-4">
+  <img src="../../media/core/imgs/pp-blinks.jpg" style="display:flex;margin:0 auto;">
+</div>
+
+:::tip
+<v-icon large color="info">info_outline</v-icon>
+See our [Best Practices](/core/best-practices/#blink-detector-thresholds) for tips on choosing 
+appropriate Blink Detector thresholds
+:::
+
+Results are exported in `blinks.csv` with the following columns:
+| Key                 | Description                                                |
+|:--------------------|:-----------------------------------------------------------|
+| `id`                | Numerical ID of blink                                      |
+| `start_timestamp`   | Blink end timestamp (ms)                                   |
+| `duration`          | Blink duration (s)                                         |
+| `end_timestamp`     | Blink start timestamp (ms)                                 |
```

Comment:
```suggestion
| `start_timestamp`   | Blink end timestamp (s)                                    |
| `duration`          | Blink duration (ms)                                        |
| `end_timestamp`     | Blink start timestamp (s)                                  |
```

Reply:
I can confirm that the unit of duration in the Blink export is seconds
[blinks.csv](https://github.com/pupil-labs/pupil-docs/files/7353132/blinks.csv)

Reply:
Riiight, blinks, not fixations. My bad.